frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -23,6 +23,15 @@ local Screen = Device.screen
 local ReaderHighlight = InputContainer:new{
 }
 
+local function bookmarkToFromLocalHighlight(bm)
+    local item = util.tableDeepCopy(bm)
+    local text = bm.notes
+    local notes = bm.text
+    item.notes = notes
+    item.text = text
+    return item
```

Comment:
Can probably be simplified: as it's a deep copy, no need for locals:
```
item.text = bm.notes
item.notes = bm.text
```
or even
`item.notes, item.text = bm.text, bm.notes`
or even
`item.notes, item.text = item.text, item.notes`

Reply:
Great, done.